lib/resolve-config.js

```diff
@@ -1,10 +1,11 @@
-const {castArray, isNil} = require('lodash');
-const urlJoin = require('url-join');
+const { castArray, isNil } = require("lodash");
```

Comment:
@b3nk3 Thanks a lot for the quick rebase! Could you please undo all the style changes here?

You can check via `npm run lint` and fix most things via `npm run lint -- --fix`

Reply:
Sorry, I had user level Prettier config and failed to catch this during rebase. I'll fix it.

Reply:
Done.